code/modules/mob/living/carbon/human/inventory.dm

```diff
@@ -39,6 +39,14 @@
 		r_store,
 		w_uniform)
 
+/mob/living/carbon/human/get_nonclothing_items() //pockets, hands, belt

```

Comment:
It won't matter, I'm taking it out in the next revision

There was a more efficient method to get what I wanted.